src/frontends/pytorch/src/frontend.cpp

```diff
@@ -176,9 +177,13 @@ void FrontEnd::normalize(const std::shared_ptr<ov::Model>& model) const {
     manager.register_pass<ov::pass::ConvertConvertLike>();
     manager.register_pass<ov::frontend::pytorch::pass::AtenIndexToSelect>();
 
+    // Mark quantized and f16/bf16 compressed constants to prevent CF for them,
+    // so that not extra memory is used for intermediate decompressed constants.
```

Comment:
Slight grammatical correction for the comments
```suggestion
    // so that no extra memory is used for intermediate decompressed constants.
```